example/app/config.yml

```diff
@@ -1,8 +1,8 @@
 
 default:
-  prefix: change-me 
```

Comment:
This should not be in the default, you should have a separate config named jl (or something).